default-config/config

```diff
@@ -37,14 +37,15 @@ InfoStoreAdd terminal xterm
 InfoStoreAdd runcmd "dmenu_run"
 InfoStoreAdd runcmdopt "-nb '#2b4e5e'"
 
-# DesktopConfiguration sets if virtual desktops/pages are "Global" 
-# This means all screens share deskstops/pages.
-# "Per-Monitor" in witch every monitor has its own virtual desktop/pages. 
-# For this mode it's recommended that FvwmPager is configured to 
-# use the monitor configuration option, so that only windows on that 
-# specific monitor are shown. Or "Shared" witch works in the same manner
-# that Xmonad or Herbsflutwm handles desks. This mode only works with 
-# >=2 monitors attached.
+# DesktopConfiguration sets how fvwm handles desktops:
+#
+# * global -- the default. This means all screens show the same desktops/pages.
+# * per-monitor -- every monitor has its own virtual desktop/pages.
+#     For this mode it's recommended that FvwmPager is configured to
+#     use the monitor configuration option, so that only windows on that
+#     specific monitor are shown.
+# * shared -- which works in the same manner that Xmonad or Herbsflutwm handles
+# desks. This mode only works with >=2 monitors attached.
 #
```

Comment:
Suggested update (it is hard to describe what this does, but maybe this improve things).

```
# * shared -- the virtual desktops are shared across all monitors and each monitor shows
#     a single desktop at a time. If one monitor switches to a desktop currently being
#     viewed by another, the two monitors swap which desktop they are viewing.
#     This is similar to manner to how Xmonad or Herbsflutwm handle desks.
#     This mode only works with 2 or more monitors.
```

Reply:
That's better.  Tweaked further....